vendor/brightray/browser/browser_main_parts.cc

```diff
@@ -186,7 +187,9 @@ void BrowserMainParts::ToolkitInitialized() {
 #endif
 
 #if defined(TOOLKIT_VIEWS)
-  views_delegate_.reset(new ViewsDelegate);
+  views_delegate_ = base::MakeUnique<ViewsDelegate>();
+ if (!views::LayoutProvider::Get())
+    layout_provider_ = ChromeLayoutProvider::CreateLayoutProvider();
```

Comment:
I think I could also use `base::MakeUnique<views::LayoutProvider>()` here instead but I wonder if doing it this way will fix some other issues we have with strange fonts on Linux.